internal/endpoints/payloads_test.go

```diff
@@ -452,3 +452,41 @@ var _ = Describe("RequestIdPayloads", func() {
 		})
 	})
 })
+
+var _ = Describe("PayloadArchiveLink", func() {
+	var (
+		handler http.Handler
+		rr      *httptest.ResponseRecorder
+
+		query map[string]interface{}
+	)
+
+	BeforeEach(func() {
```

Comment:
If possible, it might be good idea to have a happy path test for this. You might need to mock the returns from the `requestArchiveLink()` functions.

Reply:
I was thinking the same thing, a test case where everything works would be great here!

Reply:
Just added this test case. Thanks, guys!